examples/lighting-app/nrfconnect/Kconfig.sysbuild

```diff
@@ -0,0 +1,76 @@
+#
+#    Copyright (c) 2024 Project CHIP Authors
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+#
+
+#### Radio core selection
+config NRF_DEFAULT_IPC_RADIO
+	default y
+
+# Enable IEEE802.15.4 serialization to network core
+config NETCORE_IPC_RADIO_IEEE802154
+	default y if (SOC_SERIES_NRF53X) && !WIFI_NRF700X
+
+# Enable Bluetooth serialization to network core
+config NETCORE_IPC_RADIO_BT_HCI_IPC
+	default y if SOC_SERIES_NRF53X
+
+#### Bootloader
+choice BOOTLOADER
+	default BOOTLOADER_MCUBOOT
+endchoice
+
+if BOOTLOADER_MCUBOOT
```

Comment:
Should this be set somehow differently for the dongle that doesn't support DFU?

Reply:
Right, I don't see a way to add boards directory for sysbuild target itself, so I added conditions for BOARD_NRF52840DONGLE_NRF52840 related to OTA, mcuboot and factory data generation.